sdk/ai/azure-ai-generative/tests/conftest.py

```diff
@@ -0,0 +1,162 @@
+import base64
+import os
+from typing import Dict
+
+import pytest
+from devtools_testutils import (
+    FakeTokenCredential,
+    add_body_key_sanitizer,
+    add_general_regex_sanitizer,
+    add_general_string_sanitizer,
+    add_remove_header_sanitizer,
+    is_live,
+    set_custom_default_matcher,
+)
+from devtools_testutils.proxy_fixtures import EnvironmentVariableSanitizer
+
+from azure.ai.generative import AIClient
+from azure.ai.ml import MLClient
+from azure.core.credentials import TokenCredential
+from azure.identity import AzureCliCredential, ClientSecretCredential
+
+
+@pytest.fixture()
+def ai_client(
+    e2e_subscription_id: str,
+    e2e_resource_group: str,
+    e2e_project_name: str,
+    e2e_team_name: str,
+    credential: TokenCredential,
+) -> AIClient:
+    return AIClient(
+        subscription_id=e2e_subscription_id,
+        resource_group_name=e2e_resource_group,
+        team_name=e2e_project_name,
+        project_name=e2e_team_name,
```

Comment:
```suggestion
        team_name=e2e_team_name,
        project_name=e2e_project_name,
```

Reply:
Resolved by https://github.com/Azure/azure-sdk-for-python/pull/32696/commits/5034411c6f60f480318a24521328addbb65419c9